sources/api/bootstrap-containers/src/main.rs

```diff
@@ -381,26 +394,20 @@ where
 
 /// Query the API for the currently defined bootstrap containers
 async fn get_bootstrap_containers<P>(
-    socket_path: P,
+    config_path: P,
 ) -> Result<HashMap<Identifier, model::BootstrapContainer>>
 where
     P: AsRef<Path>,
 {
-    debug!("Querying the API for settings");
-
-    let method = "GET";
-    let uri = constants::API_SETTINGS_URI;
-    let (_code, response_body) = apiclient::raw_request(&socket_path, uri, method, None)
+    let config_str = tokio::fs::read_to_string(config_path.as_ref())
```

Comment:
When the API dependency is fully removed you should be able to drop the tokio dependency and all uses of async/await.